src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -397,39 +421,39 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock \
                        eclipse/che:<version> [COMMAND]
 ----
 
-To activate jpda suspend mode for debugging Che server initialization, in the `che.env`:
+To activate jpda suspend mode for debugging Che server initialization, in the `che.env` file:
 
 ----
 CHE_DEBUG_SUSPEND=true
 ----
 
-To change che debug port, in the `che.env`:
+To change the Che debug port, in the `che.env` file:
```

Comment:
verb missing after copmma

Reply:
Chnahed to "To change the Che debug port, add the following in the `che.env` file:".